src/main/java/io/openliberty/tools/intellij/liberty/lsp/ServerEnvFileType.java

```diff
@@ -43,7 +43,7 @@ private ServerEnvFileType() {
 
     @Override
     public boolean isMyFileType(@NotNull VirtualFile file) {
-        Path path = Paths.get(file.getCanonicalPath());
+        Path path = Paths.get(file.getPath());
```

Comment:
The problem with not using getCanonicalPath() here is that if someone uses a link to make a file appear to be in a directory then it will not pass the matcher in the next line. Can we just use getCanonicalPath() and check for null instead?

Reply:
Hmm, not sure how common it is for someone to symlink a server.env file, but null checking and using `getPath()` as a backup may be a good idea.

Reply:
@turkeylurkey I think it depends on which direction the symlink is going. Are you thinking of a scenario where a user opens a symlink outside of the project that is pointing to a server.env inside the project directory?

If the server.env inside the project is a symlink to a file outside the project, we actually do NOT want to resolve the symlink, as that will cause the matcher to fail (I tested this scenario).

Reply:
You're right, this is not breaking the way I thought it would.